cmd/fleetctl/package.go

```diff
@@ -286,6 +286,10 @@ func packageCommand() *cli.Command {
 				}
 			}
 
+			if opt.OsqueryDB != "" && !filepath.IsAbs(opt.OsqueryDB) {
```

Comment:
filepath will assume the current OS path separator, right? So this will not work for Windows path?

Reply:
Thanks! Great catch!

Reply:
➕

Reply:
Any ideas on how to enforce this?

My current idea is to copy https://go.dev/src/os/path_windows.go?m=text on a `pkg/filepath_windows/filepath.go` package to be used by non-Windows Go code.

Reply:
[Add filepath_windows code to check for windows paths on unix](https://github.com/fleetdm/fleet/pull/16554/commits/550913d16791aa8f44f1386749d4622c6da9677e)